src/wmnds/patterns/footer/_footer.njk

```diff
@@ -13,15 +13,43 @@
   }}
   <!-- End feedback loop -->
   {% endif -%}
-  <div class="wmnds-footer{{appClass}}">
+  <div class="wmnds-footer{{appClass}} wmnds-footer--v2">
     <div class="wmnds-container wmnds-grid">
-      <div class="wmnds-col-1 wmnds-col-md-1-2">
-        Copyright West Midlands Network &copy; 2020
+      <div class="wmnds-col-1 wmnds-col-lg-1-3 wmnds-footer__updates-col">
+        <h3 class="wmnds-footer__heading">Stay up to date</h3>
+        <p>Don’t miss out on vital information, subscribe today and be in the know at all times.</p>
+        <a href="#" class="wmnds-btn wmnds-footer__btn wmnds-footer__btn--block">Sign up to updates</a>
       </div>
-      <div class="wmnds-col-1 wmnds-col-md-1-2">
-        <a href="https://www.wmnetwork.co.uk/privacy-cookies-policy/" title="Privacy and cookies policy" target="_blank" rel="noopener noreferrer" class="wmnds-footer__link wmnds-link">Privacy &amp; Cookies Policy</a>
+      {# loop through footer menus and get items #}
+      {% for menu in params.menus -%}
+        <div class="wmnds-col-1 wmnds-col-lg-1-3">
+          <h3 class="wmnds-footer__heading wmnds-collapse-heading" aria-controls="footer-menu-{{loop.index}}" aria-expanded="true">{{menu.name}}<svg class="wmnds-collapse-heading__indicator"><use xlink:href="#wmnds-general-chevron-right" href="#wmnds-general-chevron-right"></use></svg></h3>
+          <ul class="wmnds-footer__menu wmnds-collapse-panel" id="footer-menu-{{loop.index}}">
+          {% for menuItem in menu.menuItems-%}
+            <li><a class="wmnds-footer__link" href="#">{{menuItem.name}}</a></li>
+          {% endfor -%}
+          </ul>
+        </div>
+      {% endfor -%}
+      </div>
+      <div class="wmnds-container">
+        <div class="wmnds-footer__bottom wmnds-grid">
+          <div class="wmnds-col-1 wmnds-col-md-1-2">
+            <p class="wmnds-footer__copyright">&copy; West Midlands Combined Authority 2020</p>
```

Comment:
```suggestion
            <p class="wmnds-footer__copyright">&copy; West Midlands Combined Authority <script>document.write(new Date().getFullYear())</script></p>
```
not sure if this is needed or if it is a good idea to have js in here. @daylesalmon what do you think?

Reply:
@catiarodriguescosta Yes I agree, I think this should be static text and it should be left up to the end developer who uses the footer on how they want to update the date here.

Reply:
In that case it is ready to go for me! :)